app/Member.php

```diff
@@ -22,10 +22,20 @@ class Member extends Model
      *
      * @var array
      */
-    protected $fillable = array(
```

Comment:
**Potential breakage:** Hmm, why was `$fillable` removed? It's quite different from `$visible` (which is used for JSON serialization). As per https://laravel.com/docs/5.3/eloquent#mass-assignment, will this break [usage of `Member::create` i.e. signups](https://github.com/CUMSA/members/search?utf8=%E2%9C%93&q=%22Member%3A%3Acreate%22&type=Code)? You probably want to put it back, I think?

Ideally, in a real company, we would want to have a test case for signups to make sure that we don't break the signup route - unfortunately given manpower constraints and priorities, the most prudent thing for now is probably to add a TODO (e.g. in the form of a Trello card) to remind us to write that test :(